frontend/src/components/design-system/containers/ContainerComponent.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+export interface Props {
+  padding?: 'small' | 'large';
+  background?: string;
+}
+
+withDefaults(defineProps<Props>(), {
+  padding: undefined,
```

Comment:
K cemu je tahle komponenta? Proc to ma props ktery nikde nepuzivas?

Reply:
Prostě univerzální wrapper komponenta, která řeší pozadí a padding.

Reply:
Skeleton; styly budou později.